BetterHolidays/markets/holidays.py

```
@@ -115,18 +115,28 @@ class GoodFriday(CommonHoliday):
-            day, month = tr.children[0].text.split(" ")
-            return dt.date(year, MONTHS_MAP[month], int(day[:-2]))
-        except Exception as e:  
-            raise ValueError(f"Error determining Good Friday date for {year}: {str(e)}")  
+            url = f"https://www.calendar-365.co.uk/holidays/{year}.html"
+            try:
+                html = md.HTML.from_url(url)
+            except Exception as e:
+                raise ValueError(f"Better Markdown error: {str(e)}") from e
+
+            try:
+                elements = html.inner_html.advanced_find("a", attrs={"href": f"https://www.calendar-365.co.uk/holidays/good-friday.html", "class": "link_arrow", "title": "Good Friday 2026", "text": "Good Friday"})
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)